docker/docker_client.go

```diff
@@ -357,7 +440,7 @@ func (c *dockerClient) getBearerToken(ctx context.Context, realm, service, scope
 
 // detectProperties detects various properties of the registry.
 // See the dockerClient documentation for members which are affected by this.
-func (c *dockerClient) detectProperties(ctx context.Context) error {
+func (c *dockerClient) detectProperties(ctx context.Context, isSearch bool) error {
```

Comment:
Please use a name for the parameter which describes what the parameter means/does, not what it is used for; `allowV1` or something like that.

Reply:
fixed